composer.json

```diff
@@ -22,39 +24,28 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
+        "php": "^7.3",
```

Comment:
I am not sure that we can go with php 7.3 as minimum requirement...

Reply:
there is the same dependency on ezp-ee and kernel
https://github.com/ezsystems/ezpublish-kernel/blob/master/composer.json#L10
https://github.com/ezsystems/ezplatform-ee/blob/master/composer.json#L17